README.md

```diff
@@ -71,27 +82,18 @@ Steps 5-10 are needed if you wish to use location, address or event data from th
     docker exec -it linkedevents-backend python manage.py event_import helmet --events
     # Import events from Espoo
     docker exec -it linkedevents-backend python manage.py event_import espoo --events
-8. (Optionally) import City of Helsinki internal organization for UI user rights management:
+4. import City of Helsinki internal organization for UI user rights management:
     * `docker exec -it linkedevents-backend python manage.py import_organizations https://api.hel.fi/paatos/v1/organization/ -s helsinki:ahjo`
-9. (Optionally) install API frontend templates:
+5. install API frontend templates:
     * `docker exec -it linkedevents-backend python manage.py install_templates helevents`
-10. (Optionally) import Finland municipalities and Helsinki administrative divisions, if you want to use district based API filtering of events:
+6. import Finland municipalities and Helsinki administrative divisions, if you want to use district based API filtering of events:
     ```bash
     # Import municipalities in Finland
     docker exec -it linkedevents-backend python manage.py geo_import finland --municipalities
     # Import districts in Helsinki
     docker exec -it linkedevents-backend python manage.py geo_import helsinki --divisions
     ```
 
-Finish the install:
-
-11. Start your Django server:
-    * `docker exec -it linkedevents-backend python manage.py runserver 0:8000`
-
-Now your project is live at [localhost:8000](http://localhost:8000)
-
-For UI, set authentication keys to local_settings.
-
 Installation for development
```

Comment:
Okay, since above we have the docker instructions, probably better change this headline to "Development installation on physical or virtual machine" or something similar, to clarify the difference between the two instructions? Also, since the commands are almost identical, I believe in the future these non-docker steps should be merged with the docker instructions above.

Reply:
> As regards the haystack problem, I believe the settings should be removed from local_settings.py.template, or does @vikoivun agree?

I checked https://github.com/City-of-Helsinki/linkedevents/blob/master/linkedevents/settings.py#L309 as well. I removed it in 525299a but I can revert the commit if it's needed.

> Okay, since above we have the docker instructions, probably better change this headline to "Development installation on physical or virtual machine" or something similar, to clarify the difference between the two instructions?

Done in 6ca4fb1

 > Also, since the commands are almost identical, I believe in the future these non-docker steps should be merged with the docker instructions above.

I think so too. I can do that in the future after fixing the HH importer (and the pre-reqruisites before it) 👍